examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -113,6 +114,11 @@ class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
      */
     bool IsConnected() const { return mConnectionState == CASTING_PLAYER_CONNECTED; }
 
+    /**
+     * @return true if this CastingPlayer supports the Commissioner-Generated Passcode feature.
+     */
+    bool IsCommissionerPasscodeSupported() const { return mAttributes.commissionerPasscode != 0; }
```

Comment:
It looks like the commissionerPasscode returned by the underlying SDK is a uint8\_t. To future proof the casting APIs, perhaps we should just align with that?

```suggestion
    uint8_t GetCommissionerPasscode() const { return mAttributes.commissionerPasscode; }
```

Reply:
Okay, will pass as a short to the Java layer

Reply:
Why? The spec seems to say "it is a boolean".

Reply:
I would generally try to not "change this for future proofing" unless you have clear path (i.e. happening in O(days)) that a change is needed. In 99% of the cases the "for the future" never happens and just results in more complex code.

Reply:
Per chrisdecenzo this should be a boolean. Will update.